nisystemlink/clients/feeds/_feeds_client.py

```diff
@@ -0,0 +1,101 @@
+"""Implementation of SystemLink Feeds Client."""
+
+from typing import Optional
+
+from nisystemlink.clients import core
+from nisystemlink.clients.core._uplink._base_client import BaseClient
+from nisystemlink.clients.core._uplink._methods import delete, get, post
+from uplink import Part, Path, Query
+
+from . import models
+
+
+class FeedsClient(BaseClient):
+    """Class contains a set of methods to access the APIs of SystemLink Feeds Client."""
```

Comment:
doc: Starting the docstring with "Class contains ..." or "Method performs ..." is redundant.